lib/ssl/src/dtls_handshake.erl

```diff
@@ -79,7 +79,7 @@ client_hello(_Host, _Port, Cookie, ConnectionStates,
     Extensions = ssl_handshake:client_hello_extensions(TLSVersion, CipherSuites,
                                                        SslOpts, ConnectionStates, 
                                                        Renegotiation, undefined,
-                                                       undefined),
+                                                       undefined, undefined),
```

Comment:
Do we want to implement OCSP even for DTLS or we document that it's only supported with TLS? @IngelaAndin

Reply:
There should not be anything DTLS specific for this functionality, only that it might need some
inclusion in DTLS code as initial hello handling has some differences for DTLS. So there is really no reason not to support it for DTLS as well.